src/Contrib/Jaeger/SpanConverter.php

```diff
@@ -38,12 +40,12 @@ public function convert(SpanDataInterface $span): JTSpan
         if ($span->getStatus()->getCode() !== StatusCode::STATUS_UNSET) {
             switch ($span->getStatus()->getCode()) {
                 case StatusCode::STATUS_OK:
-                    $tags[self::STATUS_CODE_TAG_KEY] = 'OK';
+                    $tags[self::STATUS_CODE_TAG_KEY] = self::STATUS_OK_WITH_ALL_UPPERCASE_LETTERS;
```

Comment:
I'd just name it `STATUS_OK` - if we learn later that it should be lower/mixed case, we can change the constant's value.

Reply:
Sure makes sense to me, updated